examples/zero-copy/programs/zero-copy/src/lib.rs

```diff
@@ -9,6 +9,41 @@ use anchor_lang::prelude::*;
 pub mod zero_copy {
     use super::*;
 
+    #[state(zero_copy)]
+    pub struct Globals {
+        pub authority: Pubkey,
+        // The solana runtime currently restricts how much one can resize an
+        // account on CPI to ~10240 bytes. State accounts are program derived
+        // addresses, which means its max size is limited by this restriction
+        // (i.e., this is not an Anchor specific issue).
+        //
+        // As a result, we only use 250 events here.
+        //
+        // For larger zero-copy data structures, one must use non-state anchor
+        // accounts, as is demonstrated below.
+        pub events: [Event; 250],
```

Comment:
it seems possible to get 2 contacts out of solana sdk and calculated that number